tslint.json

```diff
@@ -14,13 +14,7 @@
         false
       ],
       "no-console": {
-        "severity": "warning"
```

Comment:
I would change the configuration to allow only `console.error`. And we should delete the `console.log` across the board

Reply:
samesies